app/models/past_prime_ministers_index.rb

```diff
@@ -0,0 +1,10 @@
+class PastPrimeMinistersIndex < HistoricAppointmentsIndex
+  def all_prime_ministers
```

Comment:
Minor style thing:

What do you think about adding a couple of extra private methods? `prime_ministers_with_historical_accounts` and `prime_ministers_without_historical_accounts`? To make `all_prime_ministers` easier to read.